### PR TITLE
SIDM-6289 Use Warning Text to convey information about stale accounts

### PR DESCRIPTION
**Pull request by nikola-naydenov-hmcts**
Design updated:
image

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/staleUserResetPassword.jsp [hunk 1]
```diff
@@ -12,9 +12,13 @@
              <p>
                  <spring:message code="public.reset.password.stale.users.warning.fix" />
```
Comment (kremi) [inline src/main/webapp/WEB-INF/jsp/staleUserResetPassword.jsp:13]: This message is still confusing I think
"Please fix the following"
The user has nothing to fix on the page.
Should we remove it?

Comment (kremi) [reply src/main/webapp/WEB-INF/jsp/staleUserResetPassword.jsp:13]: I'm also wondering whether this needs to be in an error summary box at all?

Comment (nikola-naydenov-hmcts) [reply src/main/webapp/WEB-INF/jsp/staleUserResetPassword.jsp:13]: it can just be the message I guess, maybe UX team can weigh in

Comment (nikola-naydenov-hmcts) [reply src/main/webapp/WEB-INF/jsp/staleUserResetPassword.jsp]: design updated

**Review by kremi — COMMENTED** [review]
From the UX person:
> One other small thing: there should also be a full stop after the 'create an account' link in the second paragraph.

**Comment by nikola-naydenov-hmcts** [comment]
I added a full stop

**Review by kremi — APPROVED** [review]

**Pull request merged**